# the position of the robot
float64 pos_x
float64 pos_y

# the velocity of the robot
float64 vel_x
float64 vel_z

